Makefile

```diff
@@ -0,0 +1,105 @@
+COMPOSE := docker compose -f docker-compose.yml
+
+.PHONY: all
+all: test
+
+.PHONY: static
+static: fmt-check check check-without-features check-all-features clippy clippy-all-features clippy-cpp-rust
```

Comment:
🔧 As we requested, please extract makefile and CI-related changes to a separate PR and then base your PR on top of it.